db/migrate/20231121144340_nil_out_embed_player_urls.rb

```diff
@@ -0,0 +1,20 @@
+class NilOutEmbedPlayerUrls < ActiveRecord::Migration[7.0]
+  def up
+    Podcast.with_deleted.where("link LIKE ?", "%#{ENV["PLAY_HOST"]}%").update_all(link: nil)
+    Episode.with_deleted.where("url LIKE ?", "%#{ENV["PLAY_HOST"]}%").update_all(url: nil)
+  end
```

Comment:
Realizing we're going to get some stale caches after this.  And we don't have a cache_version yet (that's in #908).  So I guess... note to self: go nuke the redis cache when this deploys.